.github/workflows/rust.yml

```diff
@@ -18,9 +18,11 @@
 name: Rust
 
 on:
-  # always trigger
+  # always trigger on PR
   push:
   pull_request:
+  # manual trigger
+  workflow_dispatch:
```

Comment:
TIL https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch